packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -167,20 +169,15 @@ class DropdownList extends React.Component {
 		// eslint-disable-next-line react/no-find-dom-node
 		this.node = ReactDOM.findDOMNode(this);
 		Spotlight.set(this.node.dataset.spotlightId, {leaveFor: {up: '', down: ''}});
+		this.isScrolledIntoView = false;
```

Comment:
It's possible that `getDerivedStateFromProps` could assist here rather than using a class member. Something to consider in a future refactor. :)